integration-tests/test/native-eth-ovm-calls.spec.ts

```diff
@@ -203,7 +203,14 @@ describe('Native ETH value integration tests', () => {
         initialBalance,
         ValueCalls1.address
       )
-      await ValueCalls0.verifyCallValueAndSelfBalance({value: 5});
+      const balance0BeforeCall = await wallet.provider.getBalance(ValueCalls0.address)
+      console.log('balances before call:')
+      console.log([
+        await wallet.provider.getBalance(ValueCalls0.address),
+        await wallet.provider.getBalance(ValueCalls1.address)
+      ])
```

Comment:
```
balances before call:
[
  BigNumber { _hex: '0xa410', _isBigNumber: true },
  BigNumber { _hex: '0x0a', _isBigNumber: true }
]
```

Reply:
This is why @K-Ho 's previous test was failing, because the first element is not `0`